package.json

```diff
@@ -8,22 +8,18 @@
     "store",
     "behaviorsubject"
   ],
-  "main": "dist/commonjs/aurelia-store.js",
-  "typings": "dist/commonjs/aurelia-store.d.ts",
+  "main": "dist/commonjs/index.js",
+  "module": "dist/es2015/index.js",
+  "typings": "dist/index.d.ts",
   "scripts": {
     "lint": "cross-env tslint --project tsconfig.json",
     "pretest": "cross-env npm run lint",
     "test": "cross-env jest --coverage",
     "test-ci": "cross-env jest --coverage && cat ./test/coverage-jest/lcov.info | ./node_modules/coveralls/bin/coveralls.js",
     "test-watch": "jest --watch",
-    "build:amd": "cross-env tsc --p tsconfig-build.json --outDir dist/amd --module amd",
-    "build:commonjs": "cross-env tsc --p tsconfig-build.json --outDir dist/commonjs --module commonjs",
-    "build:es2015": "cross-env tsc --p tsconfig-build.json --outDir dist/es2015 --module es2015",
-    "build:es2017": "cross-env tsc --p tsconfig-build.json --outDir dist/es2017 --module es2015 --target es2017",
-    "build:native-modules": "cross-env tsc --p tsconfig-build.json --outDir dist/native-modules --module es2015",
-    "build:system": "cross-env tsc --p tsconfig-build.json --outDir dist/system --module system",
     "prebuild": "cross-env rimraf dist",
-    "build": "concurrently \"npm run build:amd\" \"npm run build:commonjs\" \"npm run build:es2015\" \"npm run build:es2017\" \"npm run build:native-modules\" \"npm run build:system\"",
+    "build": "node build/build.js",
+    "bundle-dts": "dts-bundle-generator --project tsconfig-build.json -o dist/index.d.ts src/aurelia-store.ts --external-types []",

```

Comment:
whats the externalTypes section here doing? Havent worked with the generator so would be nice to know whether we can use this to provide additional info, say from RxJS

Reply:
Somehow we are having a `/// <reference type="node" />` in generated `d.ts`, I couldn't figure out which module was causing that so I added that to work around it

Reply:
Ok ... good to know what it's used for